From 93d7fc5ee154c18a944f1305c5375c04335ed606 Mon Sep 17 00:00:00 2001
From: Liu Xiaowen <b37945@freescale.com>
Date: Thu, 7 Nov 2013 10:13:51 +0800
Subject: [PATCH 27/37] ENGR00215174 update to pass build for android 4.4.

add lost brace introduced by update.
modify the class ProcessStats import package which is
changed in android 4.4.

Signed-off-by: Liu Xiaowen <b37945@freescale.com>
---
 packages/FusedLocation/src/com/android/location/fused/FusionEngine.java | 1 +
 packages/SystemUI/src/com/android/systemui/LoadSystemTime.java          | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/packages/FusedLocation/src/com/android/location/fused/FusionEngine.java b/packages/FusedLocation/src/com/android/location/fused/FusionEngine.java
index 9588261..2eb5b15 100644
--- a/packages/FusedLocation/src/com/android/location/fused/FusionEngine.java
+++ b/packages/FusedLocation/src/com/android/location/fused/FusionEngine.java
@@ -141,6 +141,7 @@ public class FusionEngine implements LocationListener {
                 stats.minTime = minTime;
                 mLocationManager.requestLocationUpdates(name, minTime, 0, this, mLooper);
             }
+        }
         if (!stats.requested) {
             stats.requestTime = SystemClock.elapsedRealtime();
             stats.requested = true;
diff --git a/packages/SystemUI/src/com/android/systemui/LoadSystemTime.java b/packages/SystemUI/src/com/android/systemui/LoadSystemTime.java
index eb75849..37f3ec81 100644
--- a/packages/SystemUI/src/com/android/systemui/LoadSystemTime.java
+++ b/packages/SystemUI/src/com/android/systemui/LoadSystemTime.java
@@ -16,7 +16,7 @@
 
 package com.android.systemui;
 
-import com.android.internal.os.ProcessStats;
+import com.android.internal.app.ProcessStats;
 
 import android.app.Service;
 import android.content.Context;
-- 
1.8.0

